modules/1_prepare/templates/create_disk_link.sh

```diff
@@ -1,5 +1,10 @@
 #!/bin/bash
 
+if [[ -f /etc/multipath.conf ]]; then
+    sed -i 's/user_friendly_names no/user_friendly_names yes/' /etc/multipath.conf
```

Comment:
nit: We are assuming the variable will be present always which may not be true always. I'm not sure what's the default value is. Anyway we are good for now.